packages/react-router-dom/jest.config.js

```diff
@@ -1,15 +1,15 @@
 module.exports = {
-  testMatch: ["**/__tests__/*-test.js"],
+  testMatch: ["**/__tests__/*-test.[jt]s?(x)"],
   transform: {
-    "\\.js$": "./jest-transformer.js"
+    "\\.[jt]sx?$": "./jest-transformer.js",
   },
   globals: {
-    __DEV__: true
+    __DEV__: true,
   },
   moduleNameMapper: {
     "^react-router$": "<rootDir>/../../build/react-router",
     "^react-router-dom$": "<rootDir>/../../build/react-router-dom",
     "^react-router-dom\\/server$":
-      "<rootDir>/../../build/react-router-dom/server"
-  }
+      "<rootDir>/../../build/react-router-dom/server",
+  },
```

Comment:
Would you mind disabling trailing commas so we can exclude changes like this? If we enable trailing commas, I'd like to do it all at once.